examples/nivisa_dmm_measurement/measurement.py

```diff
@@ -92,10 +97,8 @@ def measure(
         instrument_type_id=INSTRUMENT_TYPE_DMM_SIMULATOR,
         timeout=RESERVATION_TIMEOUT_IN_SECONDS,
     ) as reservation:
-        resource_manager = create_visa_resource_manager(service_options.use_simulation)
-        with create_visa_session(
-            resource_manager, reservation.session_info.resource_name
-        ) as session:
+        resource_manager = create_resource_manager(service_options.use_simulation)
+        with create_session(resource_manager, reservation.session_info.resource_name) as session:
```

Comment:
FYI, we could remove another call by making `create_session`'s `resource_manager` parameter optional. However, I guess we would have to pass `use_simulation` instead.

Reply:
Updated this change for `output_voltage_measurement` , whereas updates to the `nivisa_dmm_measurement` will be taken up as part of work item 2473562.